### DISCUSSION TITLE
Instanzen lassen sich nicht als Bridge hinzufügen

### DISCUSSION THREAD

**Discussion by schudi** [thkl/hap-homematic#547] [Q&A]
Servus,
seit dem letzten Stromausfall meiner CCU3 lassen sich einige Instanzen nicht mehr in HomeKit aufnehmen.
Was auffällt ist, dass der Pin-Code auf der "Instanzen bearbeiten"-Seite mal als Link dargestellt wird (unterstrichen) und bei den Instanzen, die nicht funktionieren, nur als Text (NICHT unterstrichen).
Man kann den Code zwar trotzdem anklicken und der QR-Code erscheint, aber in HomeKit bleibt er nach dem Scannen einfach hängen mit "Mit Bridge verbinden..." und das war es.
Ich hab keine Ahnung was ich noch machen sollen, deaktivieren/reaktivieren, zurücksetzen, alles ohne Erfolg.
Es bliebe nur noch Löschen, was ich ungern machen müsste (Geräte neu hinzufügen, ...).
Hat jemand noch en Idee?
Danke und Gruß,
Michael

**Comment by timwegenerDE** [comment]
Das klingt als wäre der Port in der Firewall nicht (mehr) auf. Kannst du in der CCU mal prüfen, ob die entsprechenden Ports offen sind?

**Comment by schudi** [comment]
Servus,
Port steht in der Liste und ist auch in iptables zu sehen:
# iptables -L -vn
Chain INPUT (policy ACCEPT 2905K packets, 1254M bytes)
 pkts bytes target     prot opt in     out     source               destination
    0     0 ACCEPT     tcp  --  *      *       0.0.0.0/0            0.0.0.0/0            tcp dpt:9878
    0     0 ACCEPT     udp  --  *      *       0.0.0.0/0            0.0.0.0/0            udp dpt:9878
Aber, es scheint kein Paket zu fließen...

Auch bei netstat sieht der Port gut aus:
# netstat -tulpen | grep 9878
tcp        0      0 :::9878                 :::*                    LISTEN      2122/node

Versuche ich per telnet auf den Port zu gehen, kriege ich irgendwann "400 Bad Reqeust", klar, wenn ich nix mache.
Gruß,
M

**Reply by blackbasket** [reply]
Du sprichst am Anfang in der Mehrzahl... dann sollten es ja auch mehrere Ports sein... also nur zur Sicherheit ;) wenn du den output nur gekürzt hast, alles gut...

**Reply by schudi** [reply]
ja, war gekürzt ;-)

**Comment by timwegenerDE** [comment]
Da habe ich so spontan auch keine Idee. Das manche IDs unterstrichen sind und manche nicht habe ich auch.

Irgendwelche Änderungen im Netzwerk die in Frage kommen? Das beschriebene Verhalten (bleibt hängen bzw. bricht mit unklarem Fehler ab) hatte ich auch schon; Lag dann daran, dass ich in den Unifiy APs rumgespielt hatte und das mDNS (Multicast DNS) nicht mehr durchkam… Hast du schon mal ein anderes iOS Device versucht?
